lib/config/presets/index.ts

```diff
@@ -164,10 +164,10 @@ export async function getPreset(
   }
   const packageListKeys = [
     'description',
-    'packageNames',
-    'excludePackageNames',
-    'packagePatterns',
-    'excludePackagePatterns',
+    'matchPackageNames',
+    'matchNotPackageNames',
+    'matchPackagePatterns',
+    'matchNotPackagePatterns',
```

Comment:
Looks a little ugly to me 🤔

Reply:
So the alternative is `exclude` which makes other logic less clear. e.g. docs need to say "you must have at least one `matchX` or `excludeX`, etc. Either way introduces readability challenges, and I don't mind either way.

Reply:
ok